sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,233 @@
+# July'21 release notes
+
+Release notes for commit range 6a49170027fb..962909fe9e78
+
+## New features
+ - Implemented SYCL 2020 specialized constants [07b27965] [ba3d657] [bd8dcf4]
+   [d15b841]
+ - Provided SYCL 2020 function objects [24a2ad89]
+ - Added support for ITT notification in SYCL Runtime [a7b8daf] [8d3921e3]
+ - Implemented SYCL 2020 sub_group algorithms [e8caf6c3]
+ - Implemented SYCL 2020 `sycl::handler::host_task` method [75e5a269]
+ - Implemented SYCL 2020 `sycl::sub_group` class [19dcac79]
+ - Added support for AMD GPU devices [ec612228]
+ - Implemented SYCL 2020 `sycl::is_device_copyable` type trait [44c1cbcd]
+ - Implemented SYCL 2020 USM features [1df6873d]
+ - Implemented support for Device UUID from [Intel's Extensions for Device Information](doc/extensions/IntelGPU/IntelGPUDeviceInfo.md) [25aee287]
+ - Implemented SYCL 2020 `sycl::atomic_fence` [dcd59547]
+ - Implemented `intel::loop_count_max`, `intel::loop_count_max`,
+   `intel::loop_count_avg` attributes that allow to specify number of loop
+   iterations for FPGA [f74b4ef]
+ - Implemented generation of compiler report for kernel arguments [201f902]
+ - Implemented SYCL 2020 [sub-group](doc/extensions/SubGroup/SYCL_INTEL_sub_group.asciidoc#attributes)
```

Comment:
That is not a SYCL 2020 functionality you are referring to, that is an Intel extension